experiments:
-   id: 0
    name: "Simple Experiment"
    timeout: 10 # In seconds
    sub_actions:
    -   prefix: "trajectory"
        message: "Prepare to execute arbitrary trajectory..."
    devices:
    -   "x"
    -   "y"
    controller: "y" # "none"
    error: "none"
-   id: 1
    name: "CoppeliaSim"
    timeout: 60 # In seconds
    sub_actions:
    -   prefix: "coppelia"
        message: "human hand and IIWA7 will touch a random point in between them..."
    devices:
    -   "hand6" # lead
    -   "iiwa7" # control
    controller: "iiwa7" # "none"